engine-rest/engine-rest/src/main/java/org/camunda/bpm/engine/rest/TelemetryRestService.java

```diff
@@ -40,4 +40,8 @@
   @Produces(MediaType.APPLICATION_JSON)
   TelemetryConfigurationDto getTelemetryConfiguration();
 
+  @GET
+  @Path("/data")
+  @Produces(MediaType.APPLICATION_JSON)
+  Object getTelemetryData();
```

Comment:
For consistency, the return type could be `TelemetryDataDto` here. This would be more consistent with the other REST API interfaces. From a functional perspective, the current code is perfectly fine though.